dataflow/src/main/java/org/checkerframework/dataflow/cfg/StringCFGVisualizer.java

```diff
@@ -25,7 +20,7 @@
     @Override
     public Map<String, Object> visualize(
             ControlFlowGraph cfg, Block entry, @Nullable Analysis<V, S, T> analysis) {
-        String stringGraph = visualizeGraph(cfg, entry, analysis);
+        String stringGraph = visualizeGraph(cfg, entry, analysis).trim();
```

Comment:
Would it be hard to track down where the trailing spaces are coming from and make the output proper by construction?

Reply:
I just came up with another way to avoid trailing spaces. Thanks for the remind!